src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,30 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int getHash(K key) {
+        if (key == null) {
+            return 0;
+        }
+        return Math.abs(key.hashCode() % table.length);
+    }
+
+    private void resize() {
```

Comment:
The `resize` method is defined but not used. It should be called from the `put` method when the load factor is exceeded. Also, it should rehash all nodes from the old table to the new one.